src/System.Drawing.Common/src/System/Drawing/Design/CategoryNameCollection.cs

```diff
@@ -3,68 +3,23 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Collections;
+using System.Security.Permissions;
 
 namespace System.Drawing.Design
 {
-    /// <summary>
```

Comment:
It's my understanding we don't want doc comments like this in corefx for many reasons, the main one being that they're not used in the final product and we have the burden of maintaining 2 sets of docs

That said it wasn't deliberate that I removed these doc comments here - what do you think?